.github/workflows/pythonpackage.yml

```diff
@@ -5,48 +5,65 @@ on: [push, pull_request]
 jobs:
   Test:
     runs-on: ${{ matrix.os }}
-    env:
-        PACKAGES: "pytest flake8"
     strategy:
       fail-fast: false
-      max-parallel: 4
       matrix:
         python-version: [3.7, 3.8, 3.9]
-        gdal: [3.2.2, 3.3.0]
+        gdal: ["3.2.2", "3.3.0", "3.4.0"]
         os: [ubuntu-latest, windows-latest, macos-latest]
 
+        include:
+          - python-version: "3.10"
+            gdal: "3.4.0"
+            os: ubuntu-latest
+
+          - python-version: "3.10"
+            gdal: "3.4.0"
+            os: windows-latest
+
+          - python-version: "3.10"
```

Comment:
Are we `include`ing these combinations instead of adding 3.10 to the matrix `python-version` array because we don't expect python 3.10 to work with the other GDAL versions? Or just doesn't seem necessary to test all of them?

Reply:
Yeah, the issue is that the only GDAL builds for python 3.10 are for GDAL 3.4.0+.  So if I add `"3.10"` to `python-version`, then now I have to exclude 2 versions of GDAL across 3 OSes, which is more than I'm including.

I could very well be using the "exclude" section of the build matrix incorrectly, but I found it kind of challenging and frustrating to exclude a range of builds from the matrix.

So it seemed most direct to just state the build matrix and then include a few bonus runs for python 3.10.